guides/common/modules/proc_providing-feedback-on-red-hat-documentation.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
+
+[preface]
+[id="proc_providing-feedback-on-red-hat-documentation_{context}"]
+= Providing feedback on Red Hat documentation
+
+[role="_abstract"]
+We appreciate your input on our documentation. Please let us know how we could make it better.
+
+[l10n-exclude-start]
+* For simple comments on specific passages:
++
+. Make sure you are viewing the documentation in the _Multi-page HTML_ format. In addition, ensure you see the *Feedback* button in the upper right corner of the document.
+. Use your mouse cursor to highlight the part of text that you want to comment on.
+. Click the *Add Feedback* pop-up that appears below the highlighted text.
+. Follow the displayed instructions.
+
+[l10n-exclude-end]
```

Comment:
This looks to me it might have something to do with localization (l10n) that we use internally? But I'm wildly guessing...

Reply:
@maximiliankolb @Lennonka This must be part of the module because we translate the Satellite documentation to Japanese and we want to exclude this module from the translation (Example here: https://access.redhat.com/documentation/ja-jp/red_hat_satellite/6.11/html/installing_capsule_server/installing-capsule-server).